entries/a/app.hackthebox.com.json

```
@@ -0,0 +1,18 @@
+{
+  "Hack The Box": {
+    "domain": "app.hackthebox.com",
+    "additional-domains": [
+      "app.hackthebox.eu",
+      "www.hackthebox.com",
+      "www.hackthebox.eu"
```

subdomains shouldn't be specified for domains.
```suggestion
      "hackthebox.eu"
```